app/models/bag_man_request.rb

```diff
@@ -39,7 +39,7 @@ def cancel!(reason)
     unless cancelled
       transaction do
         update!(cancelled: true, cancel_reason: reason)
-        replication_transfer.cancel!(reason)
+        replication_transfer.cancel!(reason, last_error)
         send_cancel(replication_transfer)
       end
     end
```

Comment:
I think `send_cancel` will try to add a job on the queue, but if the rescue queue is not responding, it might fail?  Is that a realistic scenario and is it handled somehow?